src/components/ChartContainer/ChartBarDateRange.js

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
-
-function ChartBarDateRange() {
-    return (
-        
-    );
-}
-
-export default ChartBarDateRange;
+import React from 'react';
+
+function ChartBarDateRange() {
```

Comment:
It looks like this class was changed by mistake. If so it should be reverted.

Also does anyone know what this class is even doing?